cli/Cargo.toml

```diff
@@ -61,7 +61,7 @@ once_cell = "1.12.0"
 deno_runtime = { version = "0.71.0" }
 deno_core = { version = "0.145.0" }
 deno_ast = { version = "0.17.0", features = ["transpiling"] }
-birdcage = { git = "https://github.com/phylum-dev/birdcage" }
+birdcage = { git = "https://github.com/phylum-dev/birdcage", branch = "andrea/abi-v2" }
```

Comment:
We may want to hold out and merge that branch in `birdcage` before merging this.